lib/spack/spack/test/util/elf.py

```diff
@@ -17,17 +18,8 @@
 from spack.hooks.drop_redundant_rpaths import drop_redundant_rpaths
 
 
-# note that our elf parser is platform independent... but I guess creating an elf file
-# is slightly more difficult with system tools on non-linux.
-def skip_unless_linux(f):
-    return pytest.mark.skipif(
-        str(spack.platforms.real_host()) != "linux",
-        reason="implementation currently requires linux",
-    )(f)
-
-
 @pytest.mark.requires_executables("gcc")
-@skip_unless_linux
+@pytest.mark.skipif(sys.platform != "linux", reason="only tested on linux")
```

Comment:
Same. We can consolidate the fixtures and provide `linux_only`, since we now have `windows_only`